syntax = "proto3";

message User {
    int32 user_id = 1;
    string login = 2;
}

message ISearchResponse {
    repeated User found_users = 1;
    bool is_empty = 2;
}
